src/lib/components/event/event-summary-row.svelte

```diff
@@ -61,6 +65,19 @@
 
   const onLinkClick = () => {
     expanded = !expanded;
+    // Dont delete query params like ?per-page
+    const { namespace, workflow, run } = $page.params;
+    const queryParams: Record<string, string> = {};
```

Comment:
Yea, if we're doing this more than once, then we should definitely break it out.